addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -1088,6 +1088,116 @@ def test_in_invoice_line_onchange_currency_1(self):
             'amount_total': 208.01,
         })
 
+    def test_in_invoice_manual_currency_rate(self):
```

Comment:
This test is testing a lot of different things, I feel like it would be more readable if they were multiple smaller tests coming from the same original state